src/battle_ai_main.c

```diff
@@ -889,7 +889,7 @@ static s32 AI_CheckBadMove(u32 battlerAtk, u32 battlerDef, u32 move, s32 score)
                 break;
             case ABILITY_WONDER_GUARD:
                 if (effectiveness < AI_EFFECTIVENESS_x2)
-                    return 0;
+                    return 1;
```

Comment:
```suggestion
                    return RETURN_SCORE_MINUS(20);
```

Reply:
Works for me! I applied the same to the only other check that would potentially end up in similar cases, which involved Sheer Cold being used against an Ice Type. Also added the `// cannot even be selected` comment to the last remaining `return 0` that was uncommented.